continuous_integration/gpuci/build.sh

```diff
@@ -26,6 +26,12 @@ export CUDA_REL=${CUDA_VERSION%.*}
 # FIXME - monitoring GIL contention causes UCX teardown issues
 export DASK_DISTRIBUTED__ADMIN__SYSTEM_MONITOR__GIL__ENABLED=False
 
+# enable monitoring of cuDF spilling
+export CUDF_SPILL=on
+export CUDF_SPILL_STATS=1
+export DASK_DISTRIBUTED__DIAGNOSTICS__CUDF=1
```

Comment:
I initially set added this config variable to get `test_no_unnecessary_imports_on_worker[scipy]` and `test_malloc_trim_threshold` passing in GPU CI, but it seems like there isn't a trivial way to enable/disable cuDF spilling monitoring on a per-test basis.

Is there a way that we could somehow achieve this, or would it make sense to just not run these specific tests on GPU if we're expecting users to have cuDF installed on the workers?

Reply:
Would doing the same [this test](https://github.com/rapidsai/cudf/blob/7825790eac838e7a852d9b2429c64e78710cee28/python/cudf/cudf/tests/test_spilling.py#L218-L256) work? If not, maybe our only option would be to launch the test in a separate process so that we can have full control of environment variables before `import cudf`.